feed-generator/Program.cs

```diff
@@ -0,0 +1,95 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace FeedGenerator
+{
+    using Dapr.Client;
+    using Microsoft.AspNetCore.Hosting;
+    using Microsoft.Extensions.Hosting;
+    using System;
+    using System.Threading.Tasks;
+
+    /// <summary>
+    /// FeedGenerator - generates messages and publishes them using Dapr.
+    /// The main functionality is in StartMessageGeneratorAsync().
+    /// </summary>
+    public class Program
+    {
+        /// <summary>
+        /// Main for FeedGenerator
+        /// </summary>
+        /// <param name="args">Arguments.</param>
+        public static void Main(string[] args)
+        {
+            IHost host = CreateHostBuilder(args).Build();
+
+            Task.Run(() => StartMessageGeneratorAsync());
+
+            host.Run();
+        }
+
+        /// <summary>
+        /// Creates WebHost Builder.
+        /// </summary>
+        /// <param name="args">Arguments.</param>
+        /// <returns>Returns IHostbuilder.</returns>
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                .ConfigureWebHostDefaults(webBuilder =>
+                {
+                    webBuilder.UseStartup<Startup>();
+                });
+
+        static internal async void StartMessageGeneratorAsync()
+        {
+            const string PubsubTopicName = "receivemediapost";
+            TimeSpan delay = TimeSpan.FromSeconds(10);
+
+            DaprClientBuilder daprClientBuilder = new DaprClientBuilder();
+            DaprClient client = daprClientBuilder.Build();
+
+            while (true)
+            {
+                await Task.Delay(delay);
+
+                Post post = GeneratePost();
+                Console.WriteLine("Publishing");
+                await client.PublishEventAsync<Post>(PubsubTopicName, post);                
+            }
+        }
+
+        static internal Post GeneratePost()
+        {
+            Guid correlationId = Guid.NewGuid();
+            Guid messageId = Guid.NewGuid();
+            string message = GenerateRandomMessage();
+            DateTime creationDate = DateTime.UtcNow;
+
+            return new Post()
+            {
+                CorrelationId = correlationId,
+                MessageId = messageId,
+                Message = message,
+                CreationDate = creationDate
+            };
+        }
+
+        static internal string GenerateRandomMessage()
```

Comment:
I was thinking on something that looks like a text. There is a common pattern for this called Lorem Ipsum. https://stackoverflow.com/questions/4286487/is-there-any-lorem-ipsum-generator-in-c

This would make the output look like a legit message.

Reply:
yes to Lorem Ipsum

Reply:
There's no library for this and I don't know about the license for copying from SO.  While it would look nice I don't think it's the highest pri right now.